File: HelpClosing-frontend/lib/Domain/UserMailandName.dart
class UserMailandName {
  final String email;
  final String name;
  final String nickName;
  final String image;

  UserMailandName({required this.email, required this.name, required this.nickName, this.image = ''});

  factory UserMailandName.fromJson(Map<String, dynamic> json) {
    return UserMailandName(
      email: json['email'],
      name: json['name'],
      nickName: json['nickName'],
      image: json['image'],
    );
  }
}

File: HelpClosing-backend/src/main/java/cau/capstone/helpclosing/model/Response/UserMailandName.java
package cau.capstone.helpclosing.model.Response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class UserMailandName {

    private String email;
    private String name;
    private String nickName;
    private String image;
}